use-cases/eurac/trainer.py

```diff
@@ -148,6 +148,7 @@ def set_epoch(self, epoch: int):
             self.train_loader.sampler.set_epoch(epoch)
             self.val_loader.sampler.set_epoch(epoch)
 
+    @measure_gpu_utilization
```

Comment:
is it possible to use the profiler at the same time with the GPU utilization monitor?

Reply:
Yes